formulas/github-get-archive/README.md

```diff
@@ -0,0 +1,10 @@
+github-get-archive
+==================
+
+This formula adds a function that allows you to pull down a Github, Inc. repository in tar format.
```

Comment:
Capitalize H.  You can drop the `, Inc.` bit unless you're referring to the company.  I added that bit to my comment intending to say that I doubt that this arbitrary company would like their name in lowercase.